src/main/java/techcourse/myblog/model/Article.java

```diff
@@ -0,0 +1,30 @@
+package techcourse.myblog.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Getter
+@NoArgsConstructor
```

Comment:
JPA가 필요로 하는 기본 생성자는 private으로 만들어야 다른 개발자의 오용을 방지할 수 있어요. 아래의 글은 가볍게 참고만 해주세요.
https://goodyhlee.wordpress.com/2016/06/20/177

```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
```